components/using_components.rst

```diff
@@ -40,7 +40,7 @@ If you know you need a specific version of the library, add that to the command:
 
 .. code-block:: bash
 
-    $ composer require symfony/finder
+    $ composer require symfony/finder 2.6.2
```

Comment:
Then in my opinion this whole block should be removed, as the text for example is **about installing specific version of the library** while providing the same code example as the one before. And without the version composer will install the best version not the specific one.